package.json

```diff
@@ -4,24 +4,29 @@
   "description": "The documentation content home for https://docs.quantum.ibm.com.",
   "author": "Qiskit Development Team",
   "license": "Apache-2.0",
+  "type": "module",
+  "files": [
+    "dist"
+  ],
```

Comment:
This tells NPM to only include `dist/` in our package and not other files like `docs/`

Reply:
Where is `dist/`?

Reply:
I see, it's generated by `tsconfig.json`?

Reply:
Ah yes, exactly. We don't check it in to version control intentionally because we would have to manually regenerate the folder every time we make a PR.